librz/core/core.c

```diff
@@ -1,6 +1,8 @@
 // SPDX-FileCopyrightText: 2009-2020 pancake <pancake@nopcode.org>
 // SPDX-License-Identifier: LGPL-3.0-only
 
+#include "rz_mark.h"
+#include "rz_util/rz_str.h"
```

Comment:
```suggestion
#include <rz_util/rz_str.h>
```

Also in the other files.